src/utils/apolloClient.ts

```diff
@@ -0,0 +1,29 @@
+import { InMemoryCache } from 'apollo-cache-inmemory'
+import { ApolloClient } from 'apollo-client'
+import { BatchHttpLink } from 'apollo-link-batch-http'
+
+export const GIRAFFE_ENDPOINT =
+  typeof window !== 'undefined' &&
+  (window as any).GIRAFFE_ENDPOINT !== undefined
+    ? (window as any).GIRAFFE_ENDPOINT
+    : process.env.GIRAFFE_ENDPOINT || 'https://graphql.dev.hedvigit.com/graphql'
```

Comment:
Maybe we should throw if we can't find the giraffe? would be sad if we by mistake went to the dev environment in prod  😄

Reply:
Very good point, that would suck very much. Maybe a check for wether we're in a dev build? I feel like i don't wanna specify these kind of env vars every time i start a dev build of the project

Reply:
@palmfjord sounds good!

Reply:
We could maybe use this principle in giraffe as well, where the config file defaults to dev-ish values right now in lieu of envvars.